src/browser/sorterRow.html

```diff
@@ -1,27 +1,24 @@
-<tr class="ng-table-sort-header" role="row">
-  <!--
-     In an effort to provide accessible tables, if a column is sortable it should be given the aria-sort attribute 
-     with a value of 'none', 'ascending', or 'descending'. The child element should have its role set to 'button'
-  -->
+<tr class="ng-table-sort-header" role="row">  
   <!-- Checkbox template must be in a <td> as per SS-20165 -->
-  <td ng-repeat="$column in $columns" ng-init="template = $column.headerTemplateURL(this)" ng-if="$column.headerTemplateURL(this)">
+  <td ng-repeat="$column in $columns" ng-init="template = $column.headerTemplateURL(this)" ng-if="$column.show(this) && $column.headerTemplateURL(this)">
     <div ng-if="template" ng-include="template"></div>
   </td>
-  <th scope="col" title="{{$column.headerTitle(this)}}" role="columnheader" ng-attr-aria-sort="{{!$column.sortable(this) ? undefined : params.sorting()[$column.sortable(this)]=='asc' ? 'ascending' : params.sorting()[$column.sortable(this)]=='desc' ? 'descending' : 'none'}}"
-    ng-repeat="$column in $columns" ng-class="{ 
+  <th scope="col" title="{{$column.headerTitle(this)}}" role="columnheader" ng-attr-aria-sort="{{!$column.sortable(this) ? 'none' : params.sorting()[$column.sortable(this)]=='asc' ? 'ascending' : params.sorting()[$column.sortable(this)]=='desc' ? 'descending' : 'none'}}"
```

Comment:
Can we translate these strings?

Reply:
The strings for the column headers can be parsed in, in translated state when the table is instantiated on the frontend. I think this is already being done in the system.